tests/pact/db.repl

```diff
@@ -189,3 +189,56 @@
 (commit-tx)
 
```

Comment:
Need sorting testing for `keys`, `select` with backcompat testing via `(env-exec-config ["DisablePact420"])`